cypress/integration/FileInputFieldWithList/Deduplicates_the_file_list.feature

```diff
@@ -0,0 +1,7 @@
+Feature: The FileInputFieldWithList deduplicates the file list
+
+    Scenario: The file list is deduplicated
+        Given a FileInputFieldWithList with a file list including file.md is rendered
+        When the file file.md is selected
+        Then the onChange handler is called
+        Then the onChange handler's payload contains a single entry for file.md
```

Comment:
This is quite "hard" to read, I'd prefer:

```feature
Given a FileInputFieldWithList with file.md is rendered
When the file "file.md" is selected
# ...
```

Reply:
Also the next feature file says "without multiple", so I think this one needs to mention, that this one can hold multiple files, otherwise it's a bit ambiguous & this and the other feature files contradict each other.

Reply:
You could split this even more and reuse another step:

```feature
Given a FileInputFieldWithList with multiple files is rendered
And the FileInputFieldWithList holds a file "file.md"
When the file "file.md" is selected
# ...
```